src/js/common/components/RequiredLoginView.jsx

```diff
@@ -30,18 +30,16 @@ class RequiredLoginView extends React.Component {
     const userServices = this.props.userProfile.services;
     if (userServices) {
       this.setState(
-        {
-          isServiceAvailableForUse: userServices.includes(requiredApp),
-        }
+        { isServiceAvailableForUse: userServices.includes(requiredApp) }
       );
     }
   }
 
   render() {
     let view;
 
-    const loginComponent = <LoginPrompt loginUrl={this.props.loginUrl}/>;
-    const verifyComponent = <VerifyPrompt/>;
+    const loginComponent = <LoginPrompt loginUrl={this.props.loginUrl.first}/>;
+    const verifyComponent = <VerifyPrompt verifyUrl={this.props.loginUrl.third}/>;
```

Comment:
Looking over https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/reference_documents/authentication_and_authorization.md, I think this has to be 2 separate fetches